tests/xtimer_drift/Makefile

```diff
@@ -5,4 +5,19 @@ BOARD_INSUFFICIENT_MEMORY := arduino-duemilanove arduino-uno \
 
 USEMODULE += xtimer
 
+# Port and pin configuration for probing with oscilloscope
+# Define Test pin for hardware timer interrupt, hardware dependent
+# ATmega
+#CFLAGS += -DDEBUG_TIMER_PORT=PORTF
+#CFLAGS += -DDEBUG_TIMER_DDR=DDRF
+#CFLAGS += -DDEBUG_TIMER_PIN=PORTF4
+
```

Comment:
Lines 8-14: This description is already in *<cpu/atmega_common/periph/timer.c>* since #9025.
Just place a reference, please.

Reply:
@A-Paul A-Paul i am willing to remove the comments if this is the showstopper here.

Reply:
This test should be independent of the atmega implementation. Ideally, we would have a generic debug pin for this purpose configured for every platform. Now that this is out of scope for your PR, I'd say we remove it.